features/networking/service.feature

```diff
@@ -431,3 +431,48 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain "Hello OpenShift"
     """
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24694
+  @admin
+  @destructive
+  Scenario: Taint node with too small MTU value
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    And the node's MTU value is stored in the :mtu_actual clipboard
+    And evaluation of `node.name` is stored in the :subject_node clipboard
+    And I run commands on the host:
+      | systemctl stop NetworkManager                        |
+      | ip link set mtu 1300 dev <%= cb.default_interface %> |
+    Then the step should succeed
+    Given I register clean-up steps:
+    """
+    Given I use the "<%= cb.subject_node %>" node
+    And I run commands on the host:
+      | systemctl start NetworkManager |
+    Then the step should succeed
+    """
+    #This def will also store network project name in network_project_name variable
+    Given I store "<%= cb.subject_node %>" node's corresponding default networkType pod name in the :subject_node_network_pod clipboard
+    And admin ensure "<%= cb.subject_node_network_pod %>" pod is deleted from the "<%= cb.network_project_name %>" project
+    When I run the :describe admin command with:
+      | resource | node                   |
+      | name     | <%= cb.subject_node %> |
+    Then the step should succeed
+    And the output should contain "mtu-too-small"
+    #Starting NetworkManager to roll out original system MTU 
+    Given I use the "<%= cb.subject_node %>" node
+    And I run commands on the host:
+      | systemctl start NetworkManager |
+    Then the step should succeed
+    And the node's MTU value is stored in the :mtu_actual_redeployed clipboard
+    And the expression should be true> cb.mtu_actual == cb.mtu_actual_redeployed
+    #The node should get un-taint post this step in few seconds
+    And I run commands on the host:
+      | pkill openshift-sdn |
+    Then the step should succeed
+    #Check if node has removed the taint
```

Comment:
Do we need to wait some second to check the node when `pkill openshift-sdn`?

Reply:
Actually that process just get re-spawned immediately as per my testing but your comment gave me an idea to add time loop on next ```oc describe``` step in case..heh